.github/build

```diff
@@ -0,0 +1,7 @@
+tag=$(git describe --tags `git rev-list --tags --max-count=1`)
+t=${tag#"v"}
+env GOOS=linux GOARCH=amd64 go build -o clusterlint-$t-linux-amd64 ./cmd/clusterlint
```

Comment:
Given that at least for now we have our dependencies vendored, these builds will be a bit faster if we use `go build -mod=vendor` to avoid fetching dependencies on each build. (Go 1.13 defaults to ignoring `vendor/` directories and using modules).